.travis.yml

```diff
@@ -1,49 +0,0 @@
-language: scala
-scala: 2.12.15
-
-# make sure to fetch enough commits, so that git describe still works
-git:
-  depth: false
-
-env:
-  global:
-  # encrypt with: travis encrypt --com --repo playframework/play-grpc  WHITESOURCE_PASSWORD=...
-  - secure: "P+tk+59lSGZSWcsY/O6jeEKpZlXWawy3tLfGxAqs0nw/mf7OMrBVkTMnt7tcnb9ggMHIThO5CQiu+dZ4PwSY1hFFua22XVeqBQb6SWwc36ea5GY4hSHE/nHq51nShX45f3wjaHZ5rjT/jwRYmRBwvmIVFUsAExABv8YoG/I+EeIJ+JRBSAmGNPuxQ/+OUk7qAz1VrP0cUW7t68Cs66c5J+iDvQ8/UKb7u9XU43i7eoB3SnCJdyFCvAXzGkHOf3Yph0rn2dmEFIIZIxoLBx81DYQh+5efNT8pcyK5kviWIGB0XeQMTfIY3NTbaS8K8E7OLZ37Hd75c29ZHIX/7cIQOZRnnhlI2z1mCvBj6IuYnFpL3EJO+mccvTKMKuNhHmCWzTKV85wLk+1JB31ZH+mV9+hjiMY4ChiwFIbJZvBLRrHL/NqwQ0GPRQf/6GrBVydEvv2uWIB31KOnhwnie25Lwq0ptxXAzUPiwQnNJTFkbWHnrhxkabgVAKE6128SNUzUWK64c/tCjLEK8WS9WEPxVaQq1jXIYPbDadaz8yB6n9kDkg1jLkCwKCZsMumQGb/JQbhPNkYBpZG3A5WlGrAqqRU5KGbvXSlzsXi3/n06xSMaJBbZAAGY9+F9jnnYta2LxNpMcAyjLet+nXr8d/F/k59WgWcqSCuzuBjMbNtoQGI="
-  matrix:
-    - TRAVIS_JDK=8
-    - TRAVIS_JDK=11
-
-before_install: curl -Ls https://git.io/jabba | bash && . ~/.jabba/jabba.sh
-install: jabba install "adopt@~1.$TRAVIS_JDK.0-0" && jabba use "$_" && java -Xmx32m -version
-script: sbt headerCheck test:headerCheck scalafmtCheckAll scalafmtSbtCheck test paradox ++2.13.8 test
-
-jobs:
-  include:
-    - stage: publish
-      script: sbt ++2.12.15 publish ++2.13.8 "project play-grpc-generators; set publish/skip := true; project play-grpc" publish
```

Comment:
Not sure if this `set publish/skip...` should actually be configured in `build.sbt` itself. Also probably we should make sure we only call publish for the `play-grpc` project? Or maybe configure the build correctly so that only `publish` will do... I have to check tomorrow as I am to tired now...

Reply:
Note, `play-grpc` seems to be the aggregate rule.  There are several projects that look like they are published, if you run `publish` from the aggregate:
* "play-grpc-runtime"
* "play-grpc-testkit"
* "play-grpc-specs2"
* "play-grpc-scalatest"
* "lagom-javadsl-grpc-testkit"
* "lagom-scaladsl-grpc-testkit"

Many of them seem viable to _eventually_ publish.  For instance, having separate modules for test suites feels like a reasonable thing to do.  But I guess this project is still somewhat in flight.

I added the `build.play.grpc.NoPublish` to `play-grpc-generators`, since it's skipped in the travis rule.  It also feels like it could be useful to publish, but better to keep things as-is for now I guess.